src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,32 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+Port forwards activated on any `okteto up` command, independently of the dev container name.
+
+As in [forward](#forward-string-optional) in dev section, several formats are accepted for this, but it is necessary to indicate the name of the service to be forwarded using the notation `localPort:remoteService:remotePort`.
+
+```yaml
+forward:
+  - 5432:postgres:5432
+```
+
+You can also indicate the service to expose using a label selector.
+
+```yaml
+forward:
+  - localPort: 8080
+    remotePort: 80
+    name: app
+  - localPort: 5432
+    remotePort: 5432
+    labels:
+      app: db
+```
+
+Using a global forward, Okteto handles port collision if two `okteto up` sessions are running at the same time.
```

Comment:
how does Okteto handle the collision?

Reply:
if a `okteto up` session is already making use of those ports declared in the global forward section and another `okteto up` session runs, the latter will see that they are already allocated (but will be able to make use of them because the forward has already been opened by the previous session) so it will retry (loop) every so often to reopen the forward (in case the initial session terminates).

Reply:
can we include this explanation int he docs? Just saying "handles port collision" is not very useful for our users. We should mention which port takes precedence (I think is the first), and how the ports of the second okteto up will be ignored until the first one is stopped.